src/v0/sources/shopify/util.js

```diff
@@ -247,7 +249,7 @@ const checkAndUpdateCartItems = async (inputEvent, redisData, metricMetadata) =>
   const cartToken = inputEvent.token || inputEvent.id;
   if (!isDefinedAndNotNull(redisData)) {
     // eslint-disable-next-line no-param-reassign
-    redisData = await getDataFromRedis(cartToken, metricMetadata);
+    redisData = await getDataFromRedis(cartToken, metricMetadata, 'Cart Update');
```

Comment:
Why are we hard coding event name here can't we pass it as argument like others?

Reply:
Changed to take the shopifyTopic as parameter and pass updated name to metrics